aspnet/web-pages/overview/data/5-working-with-data/samples/sample27.xml

```diff
@@ -3,7 +3,7 @@
   <connectionStrings>
    <add
      name="SQLServerConnectionString"
-     connectionString= "server=myServer;database=myDatabase;uid=username;pwd=password"
+     connectionString= "server=myServer;database=myDatabase;uid=username;pwd=$CREDENTIAL_PLACEHOLDER$"
```

Comment:
```suggestion
     connectionString= "server=myServer;database=myDatabase;uid=username;$CREDENTIAL_PLACEHOLDER$"
```

Where $CREDENTIAL_PLACEHOLDER$ is the ... (not key/value pair, but use the correct term)